lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
I mean, in python we have stopped using `print` statement, and just use debugger to debug.

I think there are great debugger for the front-end too, what editor and plugin are you using?